Cloud Application Manager/Administering Your Organization/itsm-integrations.md

```diff
@@ -0,0 +1,107 @@
+
+### Prerequisites
+
+* Access to Cloud Application Manager, Management site.
+* The user must be an Administrator of the organization in Cloud Application Manager.
+
+* The user should be at the organization level scope to access the Settings option in the left side menu, or through the pencil button in the organization element in the context switcher.
+
+### Configuration Steps
+
+1. Navigate to the ITSM Integrations section in the Organization settings page and enable ServiceNow CMDB toggle:
+   ![ServiceNow CMDB toggle](../../images/cloud-application-manager/management/servicenow-cmdb-toggle.png)
+
+2. Click on the **Add ServiceNow** button:
+   ![ServiceNow CMDB add](../../images/cloud-application-manager/management/servicenow-cmdb-add.png)
+
+3. Fill in the form with the required fields:
+   ![ServiceNow CMDB add](../../images/cloud-application-manager/management/servicenow-cmdb-form.png)
+
+    Once you specify the ServiceNow host, you should copy the Redirect URL and click on the **Create an application registry for CAM** link which will open the ServiceNow server configuration in a new tab or window where you will configure the ServiceNow application registration for Cloud Application Manager (CAM) and fill in the following:
+    * Click on the lock next to Redirect URL to unlock the field and paste there the value you copied from CAM form earlier.
+    * Specify a meaningfull name for your ServiceNow application registration for CAM
+    * Copy the Client ID that has been automatically generated and go back to CAM to paste it in the corresponding field
+    * Specify the same Client Secret in both configuration pages (CAM and ServiceNow).
+    * In ServiceNow application registration page you can change the **Refresh Token Lifespan** value to "-1" to define that it will never expire.
+    * Going back to Cloud Application Manager New ServiceNow Configuration form, click on **Save**
+  ![ServiceNow CMDB save](../../images/cloud-application-manager/management/servicenow-cmdb-form-save.png)
+
+4. Once you have saved your configuration, you need to Authorize access for Cloud Application Manager to be able to talk to your ServiceNow instance by clicking the **Authorize** button:
+  ![ServiceNow CMDB authorize](../../images/cloud-application-manager/management/servicenow-cmdb-authorize.png)
+  When clicking on the **Authorize** button a ServiceNow screen will appear where you can **Allow** Cloud Application Manager to connect to your ServiceNow account.
+
+5. Once you have authorize the connection, you will see **Authorized** in your configuration, and you will have the option to **Reauthorize** in case you need to do so, or when your token expires:
+  ![ServiceNow CMDB authorized](../../images/cloud-application-manager/management/servicenow-cmdb-authorized.png)
+
+### Behavior
+
+Once ServiceNot CMDB integration has been configured, the following behavior will occur when deploying any instance through Cloud Application Manager.
```

Comment:
Please, change "Once ServiceNot" by "Once ServiceNow". Thanks!!!

Reply:
Please, change "behavior" word by "behaviour". Thanks!!!

Reply:
Thanks, @julianatienza, I addressed the first comment, but the second is expressed in American English, so it is ok as it is.
I also mentioned the Activity log.